staging/src/k8s.io/apiserver/pkg/endpoints/filters/authorization_test.go

```diff
@@ -64,14 +64,13 @@ func TestGetAuthorizerAttributes(t *testing.T) {
 
 		"resource": {
```

Comment:
While you are here, would you mind adding two test cases? @yliaog and I were chatting about this offline and we noticed that the "resource" test is actually a "create  resource without namespace" test and the "namespaced resource" test is actually a "update resource with a namespace" test. It would be more comprehensive if we also had a "create resource with namespace" test and "update resource without namespace" test.

Reply:
I have added two test cases as you suggested . In addition, I noticed that there is no test case for  subsource, so I add one.
Would you please recheck,thanks!